### ISSUE TITLE
Docker

### ISSUE THREAD

**Issue by 8thgencore** [t80tal/Instagram-Clone-Full-App#1]
Are you going dockerized project?

**Comment by Shubham-Gupta3197** [comment]
Hello @8thgencore Can I help you to dockerize your project. Please assign this issue to me.

**Comment by NehaK275** [comment]
@8thgencore Can i help you in this project?

**Comment by 8thgencore** [comment]
Yes, thanks. I want to see this project in docker using docker-compose

**Comment by NehaK275** [comment]
@8thgencore i have raised the PR. Please merge it.

**Comment by NehaK275** [comment]
Have you test it @8thgencore ?
